### PR TITLE
Split core

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,30 +2,16 @@
 members = [
   "atk",
   "atk/sys",
-  "cairo",
-  "cairo/sys",
   "gdk",
   "gdk/sys",
   "gdk-pixbuf",
   "gdk-pixbuf/sys",
```
Comment (bilelmoussaoui) [inline Cargo.toml:8]: those are part of core and not gtk3 specific

**Comment by GuillaumeGomez** [comment]
Removed gdk-pixbuf too.

**Comment by bilelmoussaoui** [comment]
Looks good to me

**Comment by GuillaumeGomez** [comment]
Merging then so I can rename the repository. Gonna send a PR to gtk4-rs afterwards.

**Comment by GuillaumeGomez** [comment]
I didn't remove the commits, simply created a mirror repository in order to keep the commit history.

**Comment by bilelmoussaoui** [comment]
Ah I see, I don't know if that makes sense but whatever I guess :)

**Pull request merged**